SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+## Supported Versions
+
+The following versions of ESMValTool are currently being supported with
+security updates.
+
+| Version  | Supported          |
+| -------- | ------------------ |
+| 2.10.0   | :white_check_mark: |
```

Comment:
This is likely to go out of date very quickly. A lower maintenance solution would be to write that only the latest release will receive security updates. If you prefer to write the version, we should start using a tool to update the version number everywhere in the code, e.g. `bumpmyversion`.

Reply:
Like this: [SECURITY.md](https://github.com/ESMValGroup/ESMValTool/blob/3455_add_security_policy/SECURITY.md)?